libvisual/tools/lv-tool/lv-tool.cpp

```diff
@@ -98,7 +98,7 @@ namespace {
   /** print info about libvisual plugin */
   void print_plugin_info(VisPluginInfo const& info)
   {
-      std::printf("Plugin: \"%s\" (%s)\n"
+      printf("Plugin: \"%s\" (%s)\n"
```

Comment:
Does the file need addition of `using namespace std;`?

PS: It would be cool if the related commit's message could explain why that change is necessary.

Reply:
I don't think it should since it should pull the c linkage version of `printf` 

I can amend the commit message

Reply:
> I don't think it should since it should pull the c linkage version of `printf`

```console
# echo $'#include <cstdio>\n\nint main() { printf("Hello\\n"); return 0; }' > main.cxx
# gcc -Wall -Wextra -pedantic -std=c++98 main.cxx
```
Seems like you're right — I was expecting some warning or so at least. Okay :smiley: 

> I can amend the commit message

As you like

Reply:
I will rebase amend them after I finalize any more changes